monitoring/metrics.rb

```diff
@@ -0,0 +1,249 @@
+require "google/cloud/monitoring"
+
+# Avoid collisions with other runs
+RANDOM_SUFFIX = Time.now.to_i.to_s
+
+def create_metric_descriptor project_id:
+  # [START monitoring_create_metric]
+  client = Google::Cloud::Monitoring::Metric.new
+  project_name = Google::Cloud::Monitoring::V3::MetricServiceClient.project_path(project_id)
+
+  descriptor = Google::Api::MetricDescriptor.new({
+    type: "custom.googleapis.com/my_metric#{RANDOM_SUFFIX}",
```

Comment:
The `RANDOM_SUFFIX` will show up in the snippet, which will be confusing for users because they don't know where it comes from, why it's there, and whether they also need it. Better not to have it.

Same in a few more places below.

Reply:
Remove RANDOM_SUFFIX and moved into method.